.github/container/install-cudnn.sh

```diff
@@ -70,3 +37,26 @@ for cudnn_file in $(dpkg -L ${libcudnn_name} ${libcudnn_dev_name} | sort -u); do
     echo "Skipping ${cudnn_file}"
   fi
 done
+
+# replicate the original symlinks too, so we'll have /opt/nvidia/cudnn/include/cudnn.sh
+find /usr/include -maxdepth 1 -name "cudnn*.h" -type l | while read -r symlink; do
+  symlink_name=$(basename "${symlink}")
+  symlink_target=$(readlink "${symlink}")
+  # Check if the symlink points to x86_64-linux-gnu/
+  if [[ "${symlink_target}" == "${arch}/"* ]]; then
+    # Adjust the symlink target to point within our symlink directory
+    adjusted_target="${prefix}/include/${symlink_target#${arch}/}"
+    # Destination symlink within the symlink directory
+    link_name="${prefix}/include/${symlink_name}"
+    link_dir=$(dirname "${link_name}")
+    mkdir -p "${link_dir}"
+    # Check if the symlink already exists
+    if [[ -e "${link_name}" ]]; then
+      echo "Symlink ${link_name} already exists. Skipping."
+    else
+      ln -s "${adjusted_target}" "${link_name}"
+    fi
+  else
+    echo "Skipping symlink ${symlink} with target ${symlink_target}"
+  fi
+done
```

Comment:
Would the link generation code above this be sufficient if `CUDNN_MAJOR_VERSION` was still set?